src/Nethermind/Nethermind.Network/ForkInfo.cs

```diff
@@ -51,6 +51,11 @@ public static ForkId CalculateForkId(ISpecProvider specProvider, long headNumber
             long next = 0;
             for (int i = 0; i < specProvider.TransitionBlocks.Length; i++)
             {
+                if (specProvider.TransitionBlocks[i] >= long.MaxValue - 100)
```

Comment:
Why this const? Can we name this const better?

Reply:
will do